docs/index.md

```diff
@@ -9,3 +9,157 @@ Learn more about using Ingress on [k8s.io](http://kubernetes.io/docs/user-guide/
 ## Getting Started
 
 See [Deployment](./deploy/) for a whirlwind tour that will get you started.
+
+
+# Migration to apiVersion networking.k8s.io/v1
+
+- Please read this [official blog on deprecated ingress api versions](https://kubernetes.io/blog/2021/07/26/update-with-ingress-nginx/) If you are using ingress objects in your pre K8s v1.22 cluster, and you upgrade to K8s v1.22, then this document may be relevant to you.
+
+- Please read this [official documentation on the IngressClass object](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class)
+
+## What is ingressClass and why is it important for users of the Ingress-Nginx-Controller now ?
+
+IngressClass is a kubernetes resource. See the description below.
+Its important because until now, a default install of the Ingress-Nginx-Controller did not require a ingressClass object. But with the new edition of the Ingress-Nginx-Controller releases, a ingressclass object is required.
```

Comment:
an ingressClass

Also, same comment about standardizing to Ingress NGINX

Reply:
Added an and changed to Ingress-NGINX